.github/workflows/pr.yml

```diff
@@ -18,6 +18,8 @@ jobs:
       run : |
         make minikube-linux-amd64
         make e2e-linux-amd64
+        make kic-base-image
```

Comment:
we should do this step only if the file changed, because rebuilding the same image, might not work because kind-base image has not pinned down the ubuntu version and each time we get some packages missing.

Reply:
I added the `--cache-from` flag to #7858 so if the Dockefile isn't changed, it'll reuse all the layers from the current image.